server/models/views.py

```diff
@@ -0,0 +1,93 @@
+from flask import (    
+    request, make_response, 
+)
+from models.recipe import Recipe
+from models.user import User
+from models.profile import Profile
+
+
+@app.route("/recipe/add/", methods=["POST"])
+def add_recipe(chef_id):
+    if request.method == "POST":
```

Comment:
you don't need this check when there's already only one method allowed on the route

Reply:
Got it. Done